opensubtitles/infos.go

```diff
@@ -0,0 +1,91 @@
+package opensubtitles
+
+import (
+	"context"
+	"net/http"
+)
+
+type InfoService service
```

Comment:
Codacy found an issue: [exported type InfoService should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)